### ISSUE TITLE
Custom kaomojis

### ISSUE THREAD

**Issue by fuzzyweapon** [jlmart88/kaomoji-slack#5]
It would be really cool if kaomoji could persist new custom kaomojis without the source needing to be changed.

- a way to add custom kaomojis
- a way to remove custom kaomojis
- a way to easily "import/export" custom kaomojis
(this can simply be providing a serialized format that kaomoji-slack understands, can spit out on 'export', can ingest and set or append on 'import', so all anyone needs to do to bootstrap another workspace with kaomojis they like is pasting it in)

Lots of ways to implement this I'm sure.  I'm not sure what these bots have access to or how their data is persisted, but this would be really nice!

**Comment by jlmart88** [comment]
That's a pretty cool idea, and shouldn't be too hard to store that data. When I get some spare time I'll take a look at what new interfaces Slack allows bots to use now, and see if there is anything that is a good fit. Feel free to open a PR as well!

**Comment by fuzzyweapon** [comment]
Yeah, if I get some time in the upcoming months to look at this (got a lot of RL stuff going on right now), I'll let you know here so we don't step all over each other.
